cmd/tools/generate/counter.yaml

```diff
@@ -45,8 +45,8 @@ counters:
   - Name: aggr_total_physical_used
     Description: Total Physical Used
 
-  - Name: aggr_volume_count_flexvol
-    Description: Number of flexvol volumes in the aggregate.
+  - Name: aggr_volume_count
+    Description: Number of volumes in the aggregate.
```

Comment:
Given the recent discussion, maybe it's better to say:
Number of volumes in the aggregate. This includes flexvols + flexgroup constituents.
or
Total number of flexvols + flexgroup constituents in the aggregate